src/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Cell from './Cell';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <div className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h2>Welcome to React Snake Game </h2>
+          <div className="board" onClick={console.log('you click board div')} onKeyDown={(e)=>{console.log(e.keyCode,'you press keydown')}}>
+            <Cell units={this.boxArray()}/>
+          </div>
+        </div>
+      </div>
+    );
+  };
+  boxArray (){
```

Comment:
Here I am trying to create the boxArray function that creates an array of boxes which is passed to from the App component to the Cell component as a prop.